### ISSUE TITLE
Ember 2.1 initializer errors

### ISSUE THREAD

**Issue by Soliah** [bdvholmes/ember-cli-mapbox#14]
Looks like more stuff has changed which causes this to break again. Specifically, `instance.container()` is undefined.

My work around for now is something like this which is ugly. My feeling is that this initializer needs to be an instance one? Not sure though, but this should be relevant: http://emberjs.com/deprecations/v2.x/#toc_initializer-arity

```
export function initialize(instance) {
  let container = this.container;

  if (!container && instance.container) {
    container = instance.container();
  } else {
    container = instance.__container__;
  }

  let config = container.lookupFactory('config:environment');

  if (!config.mapbox || !config.mapbox.accessToken) {
    console.error('Please specify your mapbox.accessToken in your config.');
    return;
  }

  L.mapbox.accessToken = config.mapbox.accessToken;
}

export default {
  name: 'mapbox-access-code',
  initialize: initialize,
};
```

**Comment by bdvholmes** [comment]
Thanks for raising @Soliah, related to #13. I'll be getting this fixed shortly.

**Comment by bdvholmes** [comment]
Fixed by #13
